runtime/crab-parachain/src/pallets/remote_governance.rs

```diff
@@ -0,0 +1,46 @@
+// --- paritytech ---
+use frame_support::traits::{EnsureOrigin, Get};
+use frame_system::RawOrigin;
+// --- darwinia-network ---
+use crate::*;
+use dp_common_runtime::remote_governance::Config;
+
+pub struct EnsureSpecific;
+impl<O: Into<Result<RawOrigin<AccountId>, O>> + From<RawOrigin<AccountId>>> EnsureOrigin<O>
+	for EnsureSpecific
+{
+	type Success = AccountId;
+
+	fn try_origin(o: O) -> Result<Self::Success, O> {
+		o.into().and_then(|o| match o {
+			RawOrigin::Signed(who)
+				if who
+					== array_bytes::hex_into_unchecked(
+						"0x129d025b24257aabdefac93d00419f06a38e3a5e2314dd6866b16e8f205ce074",
```

Comment:
This is the root account currently in use.

Reply:
Or we can register a multisig.

Reply:
Let's use the multisig address with same operators